### PR TITLE
fix(ui): cards combining into single card

### PR DESCRIPTION
**Pull request by kdsinghcoder**
issue: #86 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
 # Authentico
```
Comment (YashKumarVerma) [inline README.md:3]: no changes are required in this file

**Review by YashKumarVerma — APPROVED** [review]

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
please resolve merge conflicts.

**Review by kdsinghcoder — COMMENTED** [review]
Requested changes are implemented.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**